design-toolkit/components/src/components/menu/styles.ts

```diff
@@ -26,7 +26,7 @@ export const MenuStyles = tv({
     item: [
       'group flex items-center gap-x-s px-s text-body-s',
```

Comment:
This is my bad from the previous iteration -- we want to make sure we don't interfere with any other future "group" that app devs use, so giving this a [nested group name](https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups) is smart.

```suggestion
      'group/item flex items-center gap-x-s px-s text-body-s',
```